docs/user/parser.rst

```diff
@@ -3,6 +3,35 @@ Parsing
 The first stage of Fonduer_'s pipeline is to parse an input corpus of documents
 into the Fonduer_ data model.
 
+Fonduer supports different file formats: CSV/TSV, TXT, HTML, and hOCR.
+The diagram below illustrates how files in each format are preprocessed and consumed by :class:`Parser`.
+Nodes in dark blue represent original source files.
+You have to convert some of them into the formats that Fonduer can consume:
+a scanned document (incl. non-searchable PDF) is OCRed and exported in hOCR,
+a (born-digital) PDF is converted into hOCR using tools like pdftotree_.
+
+.. mermaid::
+
+    graph LR;
+        Scan[(Scan)]--OCR-->hOCR[(hOCR)];
+        CSV[(CSV)]-->CSVDoc;
+        TXT[(TXT)]-->TXTDoc;
+        HTML[(HTML)]-->HTMLDoc;
+        PDF[(PDF)]--Convert-->hOCR2[(hOCR)];
```

Comment:
This is kind of noted below, but should we show here that you can work with visual information using PDF+hOCR, or PDF+HTML? It seems right now to only show PDF+hOCR, unless I'm mistaken.

Reply:
Maybe just adding another line that shown PDF -- Convert --> HTML?

Reply:
You're correct. PDF+HTML is still an option.
This option is mentioned by fd88351698c3d46df7f0d0dec29b6e4fd691e3c3.